WW_FullyLeptonic/WW_FullyLeptonic.lua

```diff
@@ -0,0 +1,136 @@
+-- Load the library containing the matrix element
+load_modules('../WW_FullyLeptonic/pp_WW_fully_leptonic/build/libme_WW_ee.so')
+
+-- Global parameters used by several modules
+-- Changing these has NO impact on the value of the parameters used by the matrix element!
+parameters = {
+    energy = 13000.,
+    W_mass = 80.419002,
+    W_width = 2.047600,
+}
+
+-- Configuration of Cuba
+cuba = {
+    relative_accuracy = 0.05,
+    verbosity = 3
+}
+
+-- The transfer functions take as input the particles passed in the computeWeights() function,
+-- and each add a dimension of integration
+GaussianTransferFunction.tf_p1 = {
+    -- getpspoint() generates an input tag of type `cuba::ps_points/i`
+    -- where `i` is automatically incremented each time the function is called.
+    ps_point = getpspoint(),
+    reco_particle = 'input::particles/1',
+    sigma = 0.05,
+}
+
+GaussianTransferFunction.tf_p2 = {
+    ps_point = getpspoint(),
+    reco_particle = 'input::particles/2',
+    sigma = 0.10,
+}
+
+inputs_before_perm = {
+    'tf_p1::output',
+    'tf_p2::output',
+}
+
+
+inputs = inputs_before_perm
+
+-- Use the BreitWignerGenerators to generate values distributed as the corresponding peaks,
+-- for each propagator in the topology
+BreitWignerGenerator.flatter_s13 = {
+    ps_point = getpspoint(),
+    mass = parameter('W_mass'),
+    width = parameter('W_width')
+}
+
+BreitWignerGenerator.flatter_s24 = {
+    ps_point = getpspoint(),
+    mass = parameter('W_mass'),
+    width = parameter('W_width')
+}
+
+-- The main block defines the phase-space parametrisation,
+-- converts our particles given by the transfer functions, and our propagator masses
+-- into solutions for the missing particles in the event
+BlockF.blockf = {
+    inputs = inputs,
+
+    -- Fix the neutrino transverse momentum to the experimental MET passed to MoMEMta
+    pT_is_met = true,
+
+    s13 = 'flatter_s13::s',
+    s24 = 'flatter_s24::s',
+    q1 = getpspoint(),
+    q2 = getpspoint()
+}
+
+-- Using the fully reconstructed event (invisibles and visibles), build the initial state
+BuildInitialState.boost = {
+    particles = inputs,
+    invisibles = {
+        'blockf::invisibles',
+    },
+
+    -- Since the neutrinos were reconstructed using the experimental MET,
+    -- the event has non-zero total transverse momentum,
+    -- so we have to do the boost to define an initial state satisfying conservation
+    -- of momentum.
+    do_transverse_boost = true
```

Comment:
Since the neutrinos were NOT reconstructed using the experimental MET (see the above), you can remove these lines